extensions/wikia/ArticleVideo/bower_components/html5-skin/js/controller.js

```diff
@@ -0,0 +1,1496 @@
+      OO.log("onSkipAdClicked is called");
+      this.skin.updatePlayhead(this.state.mainVideoPlayhead, this.state.mainVideoDuration, this.state.mainVideoBuffered);
+      this.state.currentAdsInfo.skipAdButtonEnabled = false;
+      this.mb.publish(OO.EVENTS.SKIP_AD);
+      this.renderSkin();
+    },
+
+    onAdsClicked: function(source) {
+      OO.log("on ads clicked is called", source);
+      this.mb.publish(OO.EVENTS.ADS_CLICKED, {"source": source});
+    },
+
+    publishOverlayRenderingEvent: function(marginHeight) {
+      this.mb.publish(OO.EVENTS.OVERLAY_RENDERING, {"marginHeight": marginHeight});
+    },
+
+    onPlayNonlinearAd: function(event, adInfo) {
+      if(adInfo.url) {
+        this.state.adOverlayUrl = adInfo.url;
+        this.state.showAdOverlay = true;
+      }
+      this.state.pluginsElement.addClass("oo-overlay-showing");
+      var skinElement = $("#"+this.state.elementId+" .oo-player-skin");
+      var elementWidth = skinElement.width();
+      var elementHeight = skinElement.height();
+      var newCSS = {};
+      if (adInfo.ad.height && adInfo.ad.height !== -1) {
+        var padding = (adInfo.ad.paddingHeight ? adInfo.ad.paddingHeight : 0);
+        newCSS.height = (adInfo.ad.height + padding) + "px";
+        newCSS.top = "auto";
+      } else {
+        newCSS.top = 0;
+        newCSS.bottom = 0;
+      }
+      if (adInfo.ad.width && adInfo.ad.width !== -1) {
+        var padding = (adInfo.ad.paddingWidth ? adInfo.ad.paddingWidth : 0);
+        newCSS.width = (adInfo.ad.width + padding) + "px";
+        newCSS.left = "50%";
+        newCSS.transform = "translateX(-50%)"
+      }
+      this.state.pluginsElement.css(newCSS);
+      this.renderSkin();
+    },
+
+    onAdOverlayLoaded: function() {
+      this.mb.publish(OO.EVENTS.NONLINEAR_AD_DISPLAYED);
+    },
+
+    onVideoElementFocus: function(event, source) {
+      this.focusedElement = source;
+      // Make sure that the skin uses the captions that correspond
+      // to the newly focused video element
+      this.setClosedCaptionsInfo(source);
+      if (source == OO.VIDEO.MAIN) {
+        this.state.pluginsElement.removeClass("oo-showing");
+        this.state.pluginsClickElement.removeClass("oo-showing");
+      }
+    },
+
+    closeNonlinearAd: function(event) {
+      this.state.adOverlayUrl = null;
+      this.state.showAdOverlay = false;
+      this.state.showAdOverlayCloseButton = false;
+      this.state.pluginsElement.removeClass("oo-overlay-showing");
+      this.state.pluginsElement.css({
+        top: "",
+        left: "",
+        right: "",
+        bottom: "",
+        height: "0",
+        width: "0",
+        transform: ""
+      });
+      this.renderSkin();
+    },
+
+    hideNonlinearAd: function(event) {
+      this.state.showAdOverlay = false;
+      this.state.pluginsElement.removeClass("oo-overlay-showing");
+      this.renderSkin();
+    },
+
+    showNonlinearAd: function(event) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "event". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)